README.md

```diff
@@ -155,35 +155,6 @@ that `default` configuration is declared directly inside the `customs-declaratio
       "url": "http://currenturl/customs-declarations"
```

Comment:
We have File Upload related schemas in 2 places - eg for version `2.0`
```
public/api/conf/2.0/schemas/wco/declaration
public/api/conf/2.0/schemas/wco/fileupload
```

Reply:
moved to fileupload for version 2 and 3